application/models/organization.go

```diff
@@ -70,21 +70,35 @@ func (o *Organization) ValidateUpdate(tx *pop.Connection) (*validate.Errors, err
 	return validate.NewErrors(), nil
 }
 
-func (o *Organization) GetAuthProvider() (auth.Provider, error) {
+func (o *Organization) GetAuthProvider(authEmail string) (auth.Provider, error) {
```

Comment:
to satisfy go lint
```suggestion
// GetAuthProvider returns the auth provider associated with the domain of `authEmail`, if assigned, otherwise from the Organization's auth provider.
func (o *Organization) GetAuthProvider(authEmail string) (auth.Provider, error) {
```